### ISSUE TITLE
'icon' within symbolically linked directory is not supported

### ISSUE THREAD

**Issue by adamvoss** [Microsoft/vscode-vsce#177]
If the icon file exists at the specified path, but the path contains a symbolically linked directory, `vsce package` fails with:

> Error: The specified icon 'extension/img/logo.png' wasn't found in the extension.

If only the file is a symbolic link, the packaging succeeds.

vsce version: 1.22.0
OS: Linux

___
To reproduce:
1. Suppose your icon exists at **images/logo.png**
2. Execute `ln -s images img`
3. Edit **package.json** so that `icon` points to **img/logo.png**
4. Execute `vsce package`

**Comment by joaomoreno** [comment]
I suggest to stay away from symlinks during extension development.

**Comment by adamvoss** [comment]
> Are they really solving anything?

In the context this occurred, they better convey the intention of how the repository is expected to work and reduce the number of copy operations needed in preparation for packaging.  I am glad that symlinked files worked, which reduced the amount of effort needed to work around this issue.

**Comment by joaomoreno** [comment]
Got it. But you don't expect the symlinks to be preserved in the distribution package, correct?

**Comment by adamvoss** [comment]
Correct.  In my use-case, preserving symlinks would not work.  Preserving symlinks could also create portability issues.

I suppose I could understand if vsce decided to preserve symlinks, but I would expect files and directories to behave the same.  Right now file symlinks result in the target file being in included in the package.
